src/components/CardGrid/CardGrid.test.tsx

```diff
@@ -13,11 +13,11 @@ describe('<CardGrid>', () => {
       <CardGrid
         playlist={playlist}
         onCardHover={vi.fn()}
-        onCardClick={vi.fn()}
         isLoading={false}
         accessModel={'SVOD'}
         isLoggedIn={true}
         hasSubscription={true}
+        getUrl={() => '/https://test.dummy.jwplayer.com'}
```

Comment:
You can increase the test coverage a little bit by having this function take a playlist item and set the url with a param or something based on it.
```getUrl={(item: PlaylistItem) => `https://test.dummy.jwplayer.com?media_id=${item.id}`}```